root/static/scripts/place/map.js

```diff
@@ -4,7 +4,9 @@
 // http://www.gnu.org/licenses/gpl-2.0.txt
 
 const {createMap, L} = require('../common/leaflet');
-const {draggable, place, title} = require('../common/utility/getScriptArgs')();
+import getScriptArgs from '../common/utility/getScriptArgs';
```

Comment:
The presence of an import statement here causes Babel to determine that it's an ES module, which breaks the `exports.foo` assignments below (since the `exports` object is only defined in CommonJS environments). If you load the places map page you'll see those throw an exception.

The solution is to revert this or convert all the exports to ES export statements.